_data/advisory.yml

```diff
@@ -0,0 +1,119 @@
+- group: advisory
+  members:
+
+# NOTE: ONLY COMPLETED BEPs ARE INCLUDED IN THIS LIST
+#       SEE: https://bids.neuroimaging.io/get_involved.html#completed-beps
+#
+#       FURTHERMORE, THE LEADS LISTED HERE NEED NOT HAVE BEEN THE ORIGINAL BEP LEADS.
+#       RATHER, THEY MAY BE REPLACEMENTS OF THE ORIGINAL BEP LEADS WHO DID NO LONGER
+#       WANT TO BE PART OF THE BIDS ADVISORY GROUP.
+#
+#       LEADS WHO ARE INVOLVED IN SEVERAL BEPS ARE ONLY LISTED ONCE
+
+# - title: BEP001 Quantitative MRI (qMRI)
+#   leads:
+    - name: Gilles de Hollander
+      git: Gilles86
+    - name: Kirstie Whitaker
+      git: KirstieJane
+
+# - title: BEP002 BIDS Models Specification
+#   leads:
+    - name: Tal Yarkoni
+      git: tyarkoni
+
+# - title: BEP003 Common Derivatives
+#   leads:
+    - name: Chris Markiewicz
+      git: effigies
+
+# - title: BEP005 Arterial Spin Labeling (ASL)
+#   leads:
+    - name: Henk-Jan Mutsaerts
+      git: HenkMutsaerts
+    - name: Patricia Clement
+      git: patsycle
+    - name: Jan Petr
+      git: jan-petr
+    - name: Marco Castellaro
+      git: marcocastellaro
+
+# - title: BEP006 Electroencephalography (EEG)
+#   leads:
+    - name: Cyril Pernet
+      git: CPernet
+    - name: Stefan Appelhoff
+      git: sappelhoff
+    - name: Robert Oostenveld
+      git: robertoostenveld
+
+# - title: BEP007 Hierarchical Event Descriptor (HED) Tags
+#   leads:
+    - name: Kay Robbins
+      git: vislab
+    - name: Dung Truong
+      git: dungscout96
+
+# - title: BEP008 Magnetoencephalography (MEG)
+#   leads:
+    - name: Guiomar Niso
+      git: guiomar
+
+# - title: BEP009 Positron Emission Tomography (PET)
+#   leads:
+    - name: Melanie Ganz
+      git: melanieganz
+    - name: Martin Nørgaard
+      git: mnoergaard
+
+# - title: BEP010 intracranial Electroencephalography (iEEG)
+#   leads:
+    - name: Chris Holdgraf
+      git: choldgraf
+    - name: Dora Hermes
+      git: dorahermes
+
+# - title: BEP013 Resting state fMRI derivatives
+#   not included for now, as not merged into BIDS specification
+
+# - title: BEP015 Mapping file
+#   not included for now, as not merged into BIDS specification
+
+# - title: BEP018 Genetic information
+#   leads:
+    - name: Clara Moreau
+      git: claramoreau9
```

Comment:
@claramoreau9 my email to you bounced unfortunately (I seem to have an old address). May you provide me with a working address, please?